src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestAction.java

```diff
@@ -29,14 +29,17 @@
 import java.net.URL;
 import jenkins.scm.api.SCMHead;
 import jenkins.scm.api.actions.ChangeRequestAction;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.DoNotUse;
 import org.kohsuke.github.GHPullRequest;
 
 /**
- * Metadata about a {@link PullRequestSCMHead}.
+ * Retained for binary compatibility only
+ * @deprecated Retained for binary compatibility only
```

Comment:
I do not think you really mean “binary compatibility”, since this class was package access to begin with.

If you meant “settings compatibility”, are you testing upgrades of PR branch projects with `@LocalData`?